repository_service_tuf_worker/repository.py

```diff
@@ -1122,3 +1203,99 @@ def metadata_rotation(
         logging.warn(deprecation_message)
 
         return self.metadata_update(payload, update_state)
+
+    def _sign_root(self, root: Metadata[Root]) -> Tuple[bool, str]:
+        """
+        Sign the root metadata and call actions depending on the bootstrap
+        state, and if it matches the threshold
+        """
+        signed = self._validate_signatures(root, Root.type)
+        bootstrap = self._settings.get_fresh("BOOTSTRAP")
+
+        if signed and "signing" not in bootstrap:
```

Comment:
There's a problem here. `signed` may be true, even though you haven't met the threshold. Because `_validate_signatures` only check new threshold of new keys. But if you're updating root, you need to be able to sign old threshold of old keys too.

Reply:
That's true.
My new commit will limit the PR scope to enable the Distributed Asynchronous Signing (DAS) to the bootstrap only.

The metadata update will be done in another specific task/issue. It will make our interaction smaller.
You can see in https://github.com/repository-service-tuf/repository-service-tuf/issues/327